src/CalcViewModel/StandardCalculatorViewModel.cpp

```diff
@@ -216,19 +214,21 @@ void StandardCalculatorViewModel::DisplayPasteError()
     m_standardCalculatorManager.DisplayPasteError();
 }
 
-void StandardCalculatorViewModel::SetParenthesisCount(_In_ const wstring& parenthesisCount)
+void StandardCalculatorViewModel::SetParenthesisCount(_In_ unsigned int parenthesisCount)
 {
+    if (m_OpenParenthesisCount == parenthesisCount)
```

Comment:
Add curly braces around single line if statements

Reply:
👍 I should really add a rule in VS, working on many projects with different code formatting rules is confusing :D Thank you for checking!